style.css

```diff
@@ -0,0 +1,241 @@
+/* stylelint-disable no-descending-specificity */
+
+.work {
+  display: grid;
+  grid-template-rows: 1fr;
+  background-color: #e5e5e5;
+  justify-content: center;
+  margin: 0;
+}
```

Comment:
- [ ] The section should have the margins (top, bottom, left, right) and uses the font and colors (background and font) defined for the template. 
- [ ] There is a missing image for this section.
Your page
<img src="https://user-images.githubusercontent.com/65501535/133867969-46021d84-105f-4d84-a99c-2f8d5bb8a7a9.png" />
<img src="https://user-images.githubusercontent.com/65501535/133867995-e8c67bb4-9e58-41ca-9b5f-c5a392dae314.png" />

![Screenshot from 202
![Screenshot from 2021-09-18 04-40-31](https://user-images.githubusercontent.com/65501535/133867995-e8c67bb4-9e58-41ca-9b5f-c5a392dae314.png)
1-09-18 04-39-18](https://user-images.githubusercontent.com/65501535/133867969-46021d84-105f-4d84-a99c-2f8d5bb8a7a9.png)